### DISCUSSION TITLE
Unique ID's?

### DISCUSSION THREAD

**Discussion by rjm831** [spacecowboy/Feeder#333] [General]
I'm an RSS noob.  What's Generate Extra Unique IDs for and what's it do?

**Comment by spacecowboy** [comment]
> I'm an RSS noob. What's Generate Extra Unique IDs for and what's it do?

Only use it for feeds which have problems.

it should be used when the feed does not have IDs or provide faulty IDs. I have only seen one feed do it.

**Reply by rjm831** [reply]
Thanks.
